CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `StandardSerializers` now supports `bool`, `fixed32`, `uint32`, `sint32`, 
   `fixed64`, `uint64`, `sint64`, `float` and `double` primitive types, 
   `PrivateKey`, `PublicKey` and `bytes` serialization. (#514, #523)
+- Multiproofs support in `ProofMapIndexProxy`. (#507)
```

Comment:
No multiproof --- moved ^

Reply:
crap. I used github ui for resolving conflicts for the first time :)